WalletWasabi.Tests/UnitTests/Bases/ConfigManagerNgTests.cs

```diff
@@ -0,0 +1,83 @@
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using WalletWasabi.Daemon;
+using WalletWasabi.Tests.Helpers;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.Bases;
+
+/// <summary>
+/// Tests for <see cref="ConfigManagerNg"/>
+/// </summary>
+public class ConfigManagerNgTests
+{
+	/// <summary>
+	/// Tests <see cref="ConfigManagerNg.ToFile{T}(string, T)"/> and <see cref="ConfigManagerNg.LoadFile{TResponse}(string, bool)"/>.
+	/// </summary>
+	[Fact]
+	public async Task ToFileAndLoadFileTestAsync()
+	{
+		string workDirectory = await Common.GetEmptyWorkDirAsync();
+		string configPath = Path.Combine(workDirectory, $"{nameof(ToFileAndLoadFileTestAsync)}.json");
+
+		string expectedLocalBitcoinCoreDataDir = nameof(ConfigManagerNgTests);
+
+		JsonSerializerOptions options = new() { WriteIndented = true };
```

Comment:
IMO if we are not intending to use different options at different places, creation of this options could be moved inside, not to create on optional parameter only for the tests.

Inside `ConfigManagerNg` I'd do
```cs
	private static JsonSerializerOptions Options = new() { WriteIndented = true };
```
and use that to call Serializing/Deserializing, and removing `options` from params and calls.